lookout/style/format/__main__.py

```diff
@@ -149,6 +149,9 @@ def add_parser(name, help):
     eval_gen_styles_parser.add_argument(
         "reportpath", type=str,
         help="Path for report performance output.")
+    eval_gen_styles_parser.add_argument(
+        "--database", type=str, default=None,
+        help="Path to the sqlite3 database. Helps to reuse models that was trained before.")
```

Comment:
...sqlite3 database with trained models metadata. Enables to reuse previously trained models.